com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -16,17 +16,17 @@
 #endif
 
 #if defined(USING_STEREO_MATRICES)
-#define glstate_matrix_projection unity_StereoMatrixP[unity_StereoEyeIndex]
-#define unity_MatrixV unity_StereoMatrixV[unity_StereoEyeIndex]
-#define unity_MatrixInvV unity_StereoMatrixInvV[unity_StereoEyeIndex]
-#define unity_MatrixVP unity_StereoMatrixVP[unity_StereoEyeIndex]
-#define unity_MatrixInvVP mul(unity_StereoMatrixInvV[unity_StereoEyeIndex], unity_StereoCameraInvProjection[unity_StereoEyeIndex])
-
-#define unity_CameraProjection unity_StereoCameraProjection[unity_StereoEyeIndex]
-#define unity_CameraInvProjection unity_StereoCameraInvProjection[unity_StereoEyeIndex]
-#define unity_WorldToCamera unity_StereoWorldToCamera[unity_StereoEyeIndex]
-#define unity_CameraToWorld unity_StereoCameraToWorld[unity_StereoEyeIndex]
-#define _WorldSpaceCameraPos unity_StereoWorldSpaceCameraPos[unity_StereoEyeIndex]
+#define glstate_matrix_projection     unity_StereoMatrixP[unity_StereoEyeIndex]
+#define unity_MatrixV                 unity_StereoMatrixV[unity_StereoEyeIndex]
+#define unity_MatrixInvV              unity_StereoMatrixInvV[unity_StereoEyeIndex]
+#define unity_MatrixVP                unity_StereoMatrixVP[unity_StereoEyeIndex]
+#define unity_MatrixInvVP             unity_StereoMatrixIVP[unity_StereoEyeIndex]
+
+#define unity_CameraProjection        unity_StereoMatrixP[unity_StereoEyeIndex]
+#define unity_CameraInvProjection     unity_StereoMatrixIP[unity_StereoEyeIndex]
```

Comment:
Note: These are not necessarily MatrixP and MatrixIP. It might happen that in a pass like shadow cast you have a differente vp (light matrix) and still want to access camera vp. In non-stereo path we define these to always be the camera matrices and configure it once in the frame.

Reply:
unity_CameraProjection in XR is not working in URP master so there is no regression introduced by this PR.
We logged this issue to be fixed in the future PR.